modules/telco-ran-ref-design-spec.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// * telco_ref_design_specs/ran/telco-ran-ref-design-spec.adoc
+
+:_content-type: REFERENCE
+[id="telco-ran-ref-design-spec_{context}"]
+= Reference design scope
+
+The telco RAN reference design specification captures the recommended, tested, and supported configurations to get reliable and repeatable performance for a cluster running the telco RAN DU profile.
+
+[NOTE]
+====
+The validated reference configuration is updated for each major Y-stream release of {product-title}.
+Z-stream patch releases are periodically re-tested against this reference configuration including many long duration validations.
```

Comment:
Do you need to say "including many long duration validations" since you say it is periodically re-tested? If you decide to keep it, I would remove many:
```suggestion
Z-stream patch releases are periodically re-tested against this reference configuration which includes long duration validations.
```